apps/console/src/features/server-configurations/pages/connector-listing-page.tsx

```diff
@@ -0,0 +1,433 @@
+                            <div className="header-section">
+                                <Placeholder>
+                                    <Placeholder.Header>
+                                        <Placeholder.Line length="medium" />
+                                        <Placeholder.Line length="full" />
+                                    </Placeholder.Header>
+                                </Placeholder>
+                                <Divider hidden />
+                            </div>
+                        </div>
+                        <div className="content extra extra-content">
+                            <div className="action-button">
+                                <Placeholder>
+                                    <Placeholder.Line length="very short" />
+                                </Placeholder>
+                            </div>
+                        </div>
+                    </div>
+                    <Divider hidden/>
+                </div>
+            </Grid.Row>
+        );
+    };
+
+    const handleConnectorCategoryAction = (connectorCategoryId: string) => {
+        history.push(AppConstants.getPaths().get("GOVERNANCE_CONNECTORS")
+            .replace(":id", connectorCategoryId)
+        );
+    };
+
+    /**
+     * Handle connector advance setting selection.
+     */
+    const handleSelection = () => {
+        history.push(AppConstants.getPaths().get("USERNAME_VALIDATION_EDIT"));
+    };
+    
+    const handleAlternativeLoginIdentifierSelection = (): void => {
+        history.push(AppConstants.getPaths().get("ALTERNATIVE_LOGIN_IDENTIFIER_EDIT"));
+    };
+
+    const resolveConnectorCategoryIcon = (id : string): ReactElement => {
+        switch (id) {
+            case ServerConfigurationsConstants.IDENTITY_GOVERNANCE_PASSWORD_POLICIES_ID:
+                return (
+                    <PadlockAsteriskIcon className="icon" />
+                );
+            case ServerConfigurationsConstants.USER_ONBOARDING_CONNECTOR_ID:
+                return (
+                    <UserPlusIcon className="icon" />
+                );
+            case ServerConfigurationsConstants.LOGIN_ATTEMPT_SECURITY_CONNECTOR_CATEGORY_ID:
+                return (
+                    <HexagonTwoIcon className="icon" />
+                );
+            case ServerConfigurationsConstants.OTHER_SETTINGS_CONNECTOR_CATEGORY_ID:
+                return (
+                    <VerticleFilterBarsIcon className="icon" />
+                );
+            case ServerConfigurationsConstants.ACCOUNT_MANAGEMENT_CONNECTOR_CATEGORY_ID:
+                return (
+                    <CircleUserIcon className="icon" />
+                );
+            case ServerConfigurationsConstants.MFA_CONNECTOR_CATEGORY_ID:
+                return (
+                    <ShieldCheckIcon className="icon" />
+                );
+            default:
+                return null;
```

Comment:
Shall we add a default icon instead of null?

Reply:
fixed https://github.com/wso2/identity-apps/pull/4202/commits/8ab65827fa45aca316b7682aeeb2af8236bfaaa8